pages/components/button/button.css

```diff
@@ -0,0 +1,310 @@
+/* buttons */
+
+.buttons, .icons {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+/* normal button styles */
+
+.btn {
+    padding: 0.65rem;
+    border-radius: 0.25rem;
+    /* margin: 0.5rem; */
+}
+
+.primary-btn{
+  border: none;
+  color: var(--primary-text-color);
+  background: var(--primary-color-light);
+}
+
+.secondary-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--secondary-color);
+}
+
+.success-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--primary-green);
+}
+
+.danger-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--primary-red);
+}
+
+.warning-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--primary-yellow);
+}
+
+.info-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--primary-blue);
+}
+
+.light-btn{
+    border: none;
+    color: var(--primary-color-dark);
+    background: var(--primary-text-color);
+}
+
+.dark-btn{
+    border: none;
+    color: var(--primary-text-color);
+    background: var(--primary-black);
+}
+
+.link-btn{
+    border: none;
+    color: var(--dark-blue);
+    background: var(--primary-text-color);
+    text-decoration: underline
+}
+
+
+/* normal buttons hover */
+
+.primary-btn:hover {
+    background: var(--primary-color-dark);
+  }
+  
+  .secondary-btn:hover {
+    background: var(--secondary-color-dark);
+  }
+  
+  .success-btn:hover {
+    background: var(--dark-green);
+  }
+  
+  .danger-btn:hover {
+    background: var(--dark-red);
+  }
+  
+  .warning-btn:hover {
+    background: var(--dark-yellow);
+  }
+  
+  .info-btn:hover {
+    background: var(--dark-blue);
+  }
+  
+  .light-btn:hover {
+    background: var(--primary-color-lighter);
+  }
+  
+  .dark-btn:hover {
+    background: var(--primary-color);
+  }
+  
+  .link-btn:hover {
+    color: var(--dark-blue);
+  }
+
+
+
+
+/* outline button styles */
+
+.outline-primary-btn{
+    border: 1.5px solid;
+    color: var(--primary-color-dark);
+    background: transparent;
+  }
+  
+.outline-secondary-btn{
+      border: 1.5px solid;
+      color: var(--secondary-color-dark);
+      background: transparent;
+    
+  }
+  
+.outline-success-btn{
+      border: 1.5px solid;
+      color: var(--primary-green);
+      background: transparent;
+    
+  }
+  
```

Comment:
same here also you can make a class for `background: transparent`. so that you don't need to write it again and again